schema_salad/ref_resolver.py

```diff
@@ -477,6 +486,10 @@ def resolve_ref(self,
             raise ValueError(u"Expected CommentedMap or string, got %s: `%s`"
                     % (type(lref), six.text_type(lref)))
 
+        if isinstance(lref, (str, six.text_type)) and os.sep == "\\":
```

Comment:
If you're checking for it being a string, you can use ``six.string_types``
it is ``basestring`` in py27 and ``str`` in py3